Documentation/help_files/openemr_installation_help.php

```diff
@@ -0,0 +1,589 @@
+                    <h4 class="oe-help-heading"><?php echo xlt("Step 5 Configure PHP"); ?><a href="#main-list"><i class="fa fa-arrow-circle-up oe-pull-away oe-help-redirect" aria-hidden="true"></i></a></h4>
+                    <p><?php echo xlt("Clicking on 'Proceed to Step 5' in the previous step will cause the script to look at the the current system settings for PHP and will display a table with the 'Current Value' and the 'Required Value' to make OpenEMR function smoothly");?>.
+                    
+                    <p><?php echo xlt("These values are in a file called php.ini");?>.
+                    
+                    <p><?php echo xlt("The php.ini file is the default configuration file for running applications that require PHP");?>.
+                    
+                    <p><?php echo xlt("It is used to control variables such as upload sizes, file timeouts, and resource limits etc");?>.
+                    
+                    <p><?php echo xlt("The location of the php.ini file will be displayed in this step");?>.
+                    
+                    <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <?php echo xlt("NOTE: You will have to manually edit the php.ini file to enact these changes");?>.
+                    
+                    <p><?php echo xlt("It is strongly suggested that you print these instructions and edit the php.ini file at the end of the installation process");?>.
+                    
+                    <p><?php echo xlt("Setup will continue if you click on the 'Proceed to Step 6' button");?>.
+                </div>
+            </div>
+            <div class= "row" id="section7">
+                <div class="col-sm-12">
+                    <h4 class="oe-help-heading"><?php echo xlt("Step 6 Configure Apache Web Server"); ?><a href="#main-list"><i class="fa fa-arrow-circle-up oe-pull-away oe-help-redirect" aria-hidden="true"></i></a></h4>
+                    <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <?php echo xlt("NOTE: This step also has to be manually carried out");?>.
+                    
+                    <p><?php echo xlt("It will ask you to secure three directories containing patient information");?>.
+                    
+                    <p><?php echo xlt("They are"); ?>:
+                        <ul>
+                            <li><strong>/var/www/html/openemr/sites/default/documents</strong> - <?php echo xlt("where all scanned documents etc are stored"); ?></li>
+                            <li><strong>/var/www/html/openemr/sites/default/edi</strong> - <?php echo xlt("saves transmitted electronic batches for various kinds"); ?></li>
+                            <li><strong>/var/www/html/openemr/sites/default/era</strong> - <?php echo xlt("saves output from automated remittance (X12 835) processing. This includes archived remittance files, and copies of the generated HTML reports"); ?></li>
```

Comment:
Recommend customizing the paths above (if possible), like is done in the setup.php script.